### ISSUE TITLE
serenity-bdd long context value

### ISSUE THREAD

**Issue by TKlauenberg** [serenity-js/serenity-js#597]
tldr: the context value of a test is too long when using serenity-bdd report

I am using cucumber-js and protractor and the problem is shown like this: 

I have a Scenario Outline with 4 examples and for these 4 examples the serenity-bdd reporter generates one report. But in the context value, the browser and platform is added 4 times. So in my example it is "chrome,windows,chrome,windows,chrome,windows,chrome,windows". When I now try to generate the report with the serenity-bdd cli I get the error that some file could not be generated. The file has a really long filename which I think is the error there.

From my point of view I don't know if the error is in the serenity-js/serenity-bdd reporter or in the report creation in serenity-bdd itself (that the file name should be shortened there).

So first: is it desired that the context value in the report is so long and that it repeats itselft multiple times?

**Comment by jan-molak** [comment]
Hey @TKlauenberg this doesn't sound right. Would you be able to provide a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) based on [Serenity/JS Protractor Cucumber template](https://github.com/serenity-js/serenity-js-cucumber-protractor-template) so that I could try to reproduce the issue?

**Comment by TKlauenberg** [comment]
Hey @jan-molak thank you very much for the help. The `--shortFilenames` option did resolve the problem. Anyway I have [created a repository ](https://github.com/TKlauenberg/serenity-protractor-cucumber)where you can see this behaviour. The Problem seems to be a very long scenario name in Gherkin and multiple examples in a "Scenario Outline" test.

As far as my understanding of the code base is, the problem is that serenity refers to multiple tests in a Scenario Outline as one test  and then appends the Platform and Browser Tag to the context on every execution.

The Code is from the SceneReport.ts in the serenity-bdd reporter

```TypeScript
.when(BrowserTag,    (browserTag: BrowserTag) => {
    report.context   = [report.context, browserTag.browserName].filter(part => !! part).join(',');
    report.driver    = browserTag.browserName;
    report.id        = new SceneReportId(browserTag.name, report.id).value;
})
.when(PlatformTag,   (platformTag: PlatformTag) => {
    report.context   = [report.context, platformTag.name].filter(part => !! part).join(',');
    report.id        = new SceneReportId(platformTag.name, report.id).value;
})
```

**Comment by jan-molak** [comment]
Hey @TKlauenberg and thanks for helping me to reproduce the issue. It has been addressed in version 2.10.3.

**Comment by TKlauenberg** [comment]
Hey @jan-molak thank you that fixes the error 👍
